docs/documentation/full-text/term.mdx

```diff
@@ -46,4 +46,4 @@ WHERE description @@@ '(shoes running -white)';
 
 ## Special Characters
 
-The special characters `+` , `^`, ```, `:`, `{`, `}`, `"`, `[`, `]`, `(`, `)`, `~`, `!`, `\\`, `\*`, and `SPACE` must be escaped by a`\` inside the query term.
+The special characters `+` , `^`, `` ` ``, `:`, `{`, `}`, `"`, `[`, `]`, `(`, `)`, `~`, `!`, `\\`, `\*`, and `SPACE` must be escaped by a`\` inside the query term.
```

Comment:
<img width="718" alt="image" src="https://github.com/user-attachments/assets/b1df520b-5b1b-4dd6-82d0-d120c2d0b807">
I just noticed this, but is `\\` the special character that we want to highlight here, or should it be `\` (with `\\` being the escaped version).

Escaping is confusing so maybe I'm wrong here.

Reply:
I didn't modify this list, I just assumed it was correct from Ming writing it back then. Given that the other character is \*, and that we've never had a user report against it, I think it's fine

Reply:
I think it `\` might be correct here as the character that needs to be escaped? And it seems to act a bit differently in a term vs phrase:

(using the `raw` tokenizer on the `category` field that I'll be querying, so that the entire text is used as a single token, without any splitting around the special characters):
```
CALL paradedb.create_bm25_test_table(
  schema_name => 'public',
  table_name => 'mock_items'
);

CALL paradedb.create_bm25(
  index_name => 'search_idx',
  table_name => 'mock_items',
  schema_name => 'public',
  key_field => 'id',
  text_fields => paradedb.field('description') || paradedb.field('category', tokenizer=>paradedb.tokenizer('raw')),
  numeric_fields => paradedb.field('rating')
);
```

```
INSERT INTO mock_items (description, rating, category, in_stock, metadata, created_at, last_updated_date, latest_available_time)
VALUES ('item of footwear with \ char', 5, 'foot\wear', true, '{"color": "red", "size": 42}', NOW(), CURRENT_DATE, '10:00:00');

INSERT INTO mock_items (description, rating, category, in_stock, metadata, created_at, last_updated_date, latest_available_time)
VALUES ('item of footwear with double \ char', 5, 'foot\\wear', true, '{"color": "red", "size": 42}', NOW(), CURRENT_DATE, '10:00:00');
```

Term:
```
=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:foot\wear')
ORDER BY id desc LIMIT 5;
         description          | rating | category
------------------------------+--------+-----------
 item of footwear with \ char |      5 | foot\wear
(1 row)

=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:foot\\wear')
ORDER BY id desc LIMIT 5;
         description          | rating | category
------------------------------+--------+-----------
 item of footwear with \ char |      5 | foot\wear
(1 row)

=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:foot\\\wear')
ORDER BY id desc LIMIT 5;
             description             | rating |  category
-------------------------------------+--------+------------
 item of footwear with double \ char |      5 | foot\\wear
```

Phrase:
```
=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:"foot\wear"')
ORDER BY id desc LIMIT 5;
     description      | rating | category
----------------------+--------+----------
 Winter woolen socks  |      5 | Footwear
 Comfortable slippers |      3 | Footwear
 Sturdy hiking boots  |      4 | Footwear
 Generic shoes        |      4 | Footwear
 White jogging shoes  |      3 | Footwear
(5 rows)

=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:"foot\\wear"')
ORDER BY id desc LIMIT 5;
         description          | rating | category
------------------------------+--------+-----------
 item of footwear with \ char |      5 | foot\wear
(1 row)

=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:"foot\\\wear"')
ORDER BY id desc LIMIT 5;
         description          | rating | category
------------------------------+--------+-----------
 item of footwear with \ char |      5 | foot\wear
(1 row)

=> SELECT description, rating, category
FROM mock_items
WHERE id @@@ paradedb.parse('category:"foot\\\\wear"')
ORDER BY id desc LIMIT 5;
         description          | rating | category
------------------------------+--------+-----------
 item of footwear with double \ char | 5 | foot\\wear
(1 row)
```

So it looks like in term search `\` and `\\` are the same. Meaning that it is `\\` that you need to escape and type  `\\\` if you want it to be treated as a literal  double backslash.
But in phrase search `\` seems to be treated in a special way (by being ignored?) and you need to escape  it and do `\\` for it be treated as a literal slash.